package.json

```diff
@@ -6,7 +6,7 @@
   },
   "scripts": {
     "build": "lerna run build --ignore=elements-angular-example && lerna run build --scope=elements-angular-example",
-    "start": "(shx test -d ./packages/elements/dist || lerna run build --scope=@inovex.de/elements) && lerna run dev --parallel",
+    "start": "(shx test -d ./packages/elements/dist || lerna run build --scope=@inovex.de/elements) && lerna run dev --parallel --scope=@inovex.de/elements --scope=@inovex.de/elements-storybook",
```

Comment:
I changed in #660 the example-react package the command name from `build` to `serve`. What do you think is better?